docs/getting-started/gs-encryption-decryption.md

```diff
@@ -96,7 +96,7 @@ The following table describes the field layout for response decryption code.
 
 | Offset (Bytes) | Size (Bytes) | Description |
 | :--- | :--- | :--- |
-| 0 | 8 | The Unix timestamp (in milliseconds). Must be int64 big endian. |
+| 0 | 8 | The Unix timestamp (in milliseconds), in int64 big endian format. |
```

Comment:
Maybe as Andrei mentioned, we need to specify this is the timetsamp for the response:
> The Unix timestamp (in milliseconds) of the response, in int64 big endian format.